src/Components/ImagesTable/ImagesTable.js

```diff
@@ -128,6 +128,25 @@ const ImagesTable = () => {
           state: { composeRequest: compose.request, initialStep: 'review' },
         }),
     },
+    {
+      title: 'Download compose request (.json)',
```

Comment:
It's actually possible to use anchor tags in the action title, so we could do something like this if we want to avoid manipulating the DOM:
![image](https://user-images.githubusercontent.com/95542540/182816785-430f1cfb-d2b9-42fa-b4ae-8bf8afb6544f.png)

Reply:
Isn't the formatting different though? 
![image](https://user-images.githubusercontent.com/119861/182822558-e8167a7c-d8de-45ed-b921-86f4edace718.png)

Reply:
We could use css to adjust the formatting, however, it might be okay as is.

Generally we try to keep things as close to the PatternFly examples as possible. Consider this example:
https://www.patternfly.org/v4/components/table#composable-actions

The link is blue with underlining on over since it is an anchor tag. And it makes sense to use an anchor tag for this purpose, downloading a resource, in my opinion.

In any case, the design team will have the final say and we can just ask them what they want us to do!